plugins/completion/coq.nix

```diff
@@ -67,9 +67,14 @@ with lib;
         cfg.artifactsPackage
       ];
 
+      globals = {
+        coq_settings = cfg.settings;
+      };
+
+      extraConfigLua = "require('coq')";
+
       plugins.lsp = {
         preConfig = ''
-          vim.g.coq_settings = ${helpers.toLuaObject cfg.settings}
           local coq = require 'coq'
```

Comment:
This is superflous now, isn't it?

Reply:
it is needed for use in "setupWrappers" as it references that variable